src/compiler/types.ts

```diff
@@ -1525,9 +1539,7 @@ namespace ts {
         getConstantValue(node: EnumMember | PropertyAccessExpression | ElementAccessExpression): number;
         getBlockScopedVariableId(node: Identifier): number;
         getReferencedValueDeclaration(reference: Identifier): Declaration;
-        serializeTypeOfNode(node: Node): string | string[];
-        serializeParameterTypesOfNode(node: Node): (string | string[])[];
-        serializeReturnTypeOfNode(node: Node): string | string[];
+        isTypeWithValue(node: TypeReferenceNode): TypeWithValueResolutionResult; 

```

Comment:
'is' function returning a non-boolean result.  Smells weird.  this seems more like getTypeWithValueKind(...), or something like that.